distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1392,6 +1392,76 @@ type Table
         msg = "Parsing values is not supported in database tables, the table has to be materialized first with `read`."
         Error.throw (Unsupported_Database_Operation.Error msg)
 
+    ## Splits a column of text into a set of new columns.
+       The original column will be removed from the table.
+       The new columns will be named with the name of the input with a
+       incrementing number after.
+
+       Arguments:
+       - column: The column to split the text of.
+       - delimiter: The term or terms used to split the text.
+       - column_count: The number of columns to split to.
+         If `Auto` then columns will be added to fit all data.
+         If the data exceeds the `column_count`, a `Column_Count_Exceeded` error
+         will follow the `on_problems` behavior.
```

Comment:
```suggestion
         If the data exceeds the `column_count`, a `Column_Count_Exceeded` will be
         reported according to the `on_problems` behavior.
```

I think we try to not use the 'error' term here, as it may be just a warning too.

Maybe we should move this to a section `! Error Conditions` where we could describe all kinds of issues this function may encounter, like we started doing for many of the table operations.